### PR TITLE
docs: Minor corrections and updates to Cohere docs

### PR DESCRIPTION
**Pull request by Anirudh31415926535**
- **Description:** Update the Cohere's provider and RagRetriever documentations with latest updates.
    - **Twitter handle:** Anirudh1810

### CODE REVIEW

**Review by baskaryan — APPROVED** [review]

File: docs/docs/integrations/providers/cohere.mdx [hunk 1]
```diff
@@ -46,6 +46,55 @@ print(llm.invoke("Come up with a pet name"))
 ```
 
 Usage of the Cohere (legacy) [LLM model](/docs/integrations/llms/cohere) 
+
+### Tool calling
```
Comment (baskaryan) [inline docs/docs/integrations/providers/cohere.mdx:50]: could we add this to the chat model doc instead docs/docs/integrations/chat/cohere.ipynb

Comment (maximevoisincohere) [reply docs/docs/integrations/providers/cohere.mdx:50]: hey @baskaryan, is there any downside to add it in both places?

Comment (Anirudh31415926535) [reply docs/docs/integrations/providers/cohere.mdx:50]: Hey @baskaryan I've added it to both the chat model doc as well as provider. Do lmk if there are any concerns in doing that.

**Comment by baskaryan** [comment]
need to lint docs
```bash
poetry install --with lint
make format
make lint
```

**Pull request merged**